File: official/guides/partner-white-label-guide/java/fund-wallet.java
package referral;

import com.easypost.service.EasyPostClient;
import com.easypost.model.PaymentMethod;
import com.easypost.exception.EasyPostException;

public class RetrieveReferralUsers {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        client.billing.fundWallet("2000", PaymentMethod.Priority.PRIMARY);
    }
}

File: official/guides/partner-white-label-guide/csharp/fund-wallet.cs
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            await client.Billing.FundWallet("20.00", PaymentMethod.Priority.Primary);
        }
    }
}
